package.json

```diff
@@ -301,6 +301,12 @@
           "type": ["integer", "null"],
           "default": null
         },
+        "ruff.prioritizeFileConfiguration": {
+          "markdownDescription": "Whether to prioritize file configuration over configuration set in extension settings.",
+          "scope": "window",
+          "type": "boolean",
+          "default": false
+        },
```

Comment:
I wonder if this should be a dropdown instead. I could see multiple options

* Default: use the editor settings only if the project doesn't have a configuration file
* Override: Use the project settings but allow the editor settings to override individual settings
* Fallback: use the editor settings as fallbacks when the corresponding keys aren't set in the settings file.

I'm not very fond of the names and we don't need to implement all three variants today, but I think adding a dropdown gives us more flexibility in the future.

Reply:
I agree that a dropdown might be a better choice here. I'll update the `ruff` PR to reflect this as well.